### ISSUE TITLE
Collapsible elements in a row are broken

### ISSUE THREAD

**Issue by dkanthofer** [payloadcms/payload#3778]
### Link to reproduction

N/A

### Describe the Bug

When placing two collapsible elements next to each other inside a row element, the collapsible elements are not aligned correctly.

<img width="902" alt="image" src="https://github.com/payloadcms/payload/assets/3141586/ca6a987e-0f1e-48a8-aac6-7d774fd5e6d6">

### To Reproduce

Use this field definition in the collection config:

```{
      type: "row",
      fields: [
        {
          type: "collapsible",
          label: "Collapsible 1",
          fields: [
            {
              name: "field1",
              label: "Access",
              type: "select",
              required: false,
              options: [
                {
                  label: "Value 1",
                  value: "val1",
                },
                {
                  label: "Value 2",
                  value: "val2",
                },
              ],
            },
            {
              name: "field2",
              type: "text",
              localized: true,
            },
          ],
        },
        {
          type: "collapsible",
          label: "Collapsible 1",
          fields: [
            {
              name: "field3",
              type: "select",
              options: [
                {
                  label: "Value 1",
                  value: "val1",
                },
                {
                  label: "Value 2",
                  value: "val2",
                },
              ],
            },
            {
              name: "field4",
              type: "text",
              localized: true,
            },
          ],
        },
      ],
    },```

### Payload Version

2.0.11

### Adapters and Plugins

db-mongodb, bundler-webpack

**Comment by ajaysehwal** [comment]
i am start working on this issue

**Comment by jacobsfletch** [comment]
Hey @dkanthofer this has been resolved with #3822 and will go out in the next release. Thank you for the report 🙏 and @ajaysehwal looking forward to some more contributions from you 🙌
